build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-make -j T=open_source DEBUG=1 >log.txt 2>&1
+make T=open_source DEBUG=1 >log.txt 2>&1
```

Comment:
Can you revert the removal of the `-j`

Reply:
Yup! Good catch - thought I'd pushed that. Build is still runs successfully with the flag re-added to `build.sh`